store/db_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"database/sql"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+	"sync"
+	"testing"
+)
+
+func insertValue(db *DB, wg *sync.WaitGroup, t *testing.T) {
```

Comment:
There are some problems with this function.

- I don't like the fact that this function is taking some hodgepodge of parameters. It should just take a `db *DB` and return an error, IMO.

- This function is called inside a goroutine, so we cannot just call `t.Fatalf` here. It will kill the goroutine, but not the entire test case. Please have a look at the [goroutine assistant](https://github.com/coreos/rkt/blob/master/tests/testutils/goroutineassistant.go) and how it is used in [network tests](https://github.com/coreos/rkt/blob/master/tests/rkt_net_test.go).